main.py

```diff
@@ -727,6 +749,28 @@ def draw_menu(self):
             # Escape option
             pyxel.text(26, 95, "Press SPACE to return", 1)
             pyxel.text(25, 95, "Press SPACE to return", 7)
+        elif self.menu_skin:
+            # Choose the skin...
+            # Intro text
+            pyxel.text(16, 25, "=== Choose A Skin Pack ===", 1)
+            pyxel.text(15, 25, "=== Choose A Skin Pack ===", 7)
+            # Skin A
+            pyxel.text(
+                26, 35, "[1] Skin A" + ("<-" if self.current_skin == 0 else ""), 1
+            )
+            pyxel.text(
+                25, 35, "[1] Skin A" + ("<-" if self.current_skin == 0 else ""), 7
+            )
+            # Skin B
+            pyxel.text(
+                26, 45, "[2] Skin B" + ("<-" if self.current_skin == 1 else ""), 1
+            )
+            pyxel.text(
+                25, 45, "[2] Skin B" + ("<-" if self.current_skin == 1 else ""), 7
+            )

```

Comment:
Just a small style fix:

```suggestion
            pyxel.text(
                26, 35, "[1] Skin A" + (" <-" if self.current_skin == 0 else ""), 1
            )
            pyxel.text(
                25, 35, "[1] Skin A" + (" <-" if self.current_skin == 0 else ""), 7
            )
            # Skin B
            pyxel.text(
                26, 45, "[2] Skin B" + (" <-" if self.current_skin == 1 else ""), 1
            )
            pyxel.text(
                25, 45, "[2] Skin B" + (" <-" if self.current_skin == 1 else ""), 7
            )

```

Reply:
Ok small spacing is good